SkillChat.Client.ViewModel/MessageCleaningViewModel.cs

```diff
@@ -0,0 +1,47 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Windows.Input;
+using PropertyChanged;
+using ReactiveUI;
+using SkillChat.Interface;
+using Splat;
+
+namespace SkillChat.Client.ViewModel
+{
+    [AddINotifyPropertyChangedInterface]
+    public class MessageCleaningViewModel
```

Comment:
Думаю что это окно стало таким универсальным, что его можно переименовать в ConfirmationViewModel
```suggestion
    public class ConfirmationViewModel
```

Reply:
Сделал